### PR TITLE
Add scheduled releases

### PR DESCRIPTION
**Pull request by mdo**
Not sure if this belongs here or somewhere else, but it'd be rad to plan things out more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,20 @@ Versions in _Maintenance_ should not have any changes landed, except for:
...
+
+- **Bootstrap 4:**
+  - v4.5.3 in September 2020
+  - v4.6.0 in October 2020
+- **Bootstrap 5:**
+  - v5.0.0-alpha2 in September 2020
+  - v5.0.0-alpha3 in October 2020
+  - v5.0.0-beta1 in October 2020
+  - v5.0.0-beta2 in November 2020
+  - v5.0.0 in December 2020
```
Comment (mdo) [inline README.md:52]:
```suggestion
- **Bootstrap 4:**
  - v4.5.3 in September 2020 (bug fixes and docs)
  - v4.6.0 in October 2020 (feature alignment with v5)
  - v4.6.x in November 2020 (bug fixes and docs)
- **Bootstrap 5:**
  - v5.0.0-alpha2 in September 2020 (delayed due to team vacations)
  - v5.0.0-alpha3 in October 2020 (additional new features and breaking changes)
  - v5.0.0-beta1 in October 2020 (final instance of breaking changes)
  - v5.0.0-beta2 in November 2020 (beta bug fixes, possibly a 3rd beta)
  - v5.0.0 in December 2020 (stable release)
```

Comment (mdo) [reply README.md:52]: Or with a table?

```suggestion
| v4 Releases | Target date | Description |
|:--- |:--- |:--- |
| v4.5.3 | September 2020 | Bug fixes and docs updates |
| v4.6.0 | October 2020   | Continued feature alignment with v5 |
| v4.6.x | November 2020  | Bug fixes and docs updates |

| v5 Releases | Target date | Description |
|:--- |:--- |:--- |
| v5.0.0-alpha2 | September 2020 | Features and fixes, delayed due to team vacations |
| v5.0.0-alpha3 | September 2020 | Additional new features and breaking changes |
| v5.0.0-beta1  | September 2020 | Final instance of breaking changes |
| v5.0.0-beta2  | September 2020 | Beta bug fixes, with contingency for additional quick betas |
| v5.0.0        | September 2020 | Stable release |
```

Comment (MartijnCuppens) [reply README.md:52]: I'm ok with the version I 👍 'ed

**Comment by mdo** [comment]
Ready for review @MartijnCuppens @XhmikosR.

**Comment by MartijnCuppens** [comment]
LGTM. We should remove the current graph which is quite misleading

**Comment by XhmikosR** [comment]
Yeah, agreed. I couldn't figure out how to automatically generate the SVG image properly.

**Comment by XhmikosR** [comment]
https://github.com/twbs/release/compare/release-schedule...XhmikosR-patch-1

**Comment by XhmikosR** [comment]
We should also update the table data and if we don't plan to maintain https://github.com/twbs/release/blob/master/schedule.json drop that, otherwise update that too.

**Comment by mdo** [comment]
Pulled in @XhmikosR's changes. Also took a rough stab at updating dates (setting them further out largely for placeholder, but also trying to set some expectations). Unsure how we update the SVG, or if we just drop it?

**Comment by mdo** [comment]
Removed the SVG and JSON since we pulled those from the readme.

**Review by MartijnCuppens — APPROVED** [review]

**Review by XhmikosR — APPROVED** [review]

**Comment by captn3m0** [comment]
Since this is no longer maintained, perhaps the `master` branch could be deleted as well? I had the [image hotlinked on endoflife.date/bootstrap] at https://raw.githubusercontent.com/twbs/release/master/schedule.svg?sanitize=true, and that still serves the image.

Just noticed that it was outdated today on the v5 release.

**Pull request merged**